app/views/steps/miam/certification_date/edit.html.erb

```diff
@@ -20,3 +20,5 @@
     <% end %>
   </div>
 </div>
+
+# TODO: Sort out making the hint text visually unhidden when the legend is visually hidden
```

Comment:
We can't leave this comment here, as it will be visible in the page.
We can change it to an HTML comment `<!-- blah --> `

Reply:
I have removed this :)